src/app/components/BlockNavigationButtons/index.tsx

```diff
@@ -44,11 +45,13 @@ export const NextBlockButton: FC<{ scope: SearchScope; currentRound: number }> =
   return (
```

Comment:
`const disabled = currentRound >= latestBlock`
depending on react-query config, it could return slightly stale runtime status result

Reply:
OK, done